### PR TITLE
new: Add playlist settings

### PR DESCRIPTION
**Pull request by ras1**

### CODE REVIEW

**Review by hebron-george — APPROVED** [review]
Lgtm

**Comment by hebron-george** [comment]
Is there a front end change to go with this?

**Comment by ras1** [comment]
> Is there a front end change to go with this?

Not yet

**Comment by ras1** [comment]
@hebron-george
Was able to fix CI by updating ruby to 2.6.6 + updating Gemfile.lock. ruby 2.5.1 had an issue in pipeline around this: 
https://bundler.io/blog/2019/05/14/solutions-for-cant-find-gem-bundler-with-executable-bundle.html

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.5.1'
+ruby File.open('.ruby-version', 'rb') { |f| f.read.chomp }
```
Comment (hebron-george) [inline Gemfile:4]: nice, so we don't have to remember to update anymore

**Review by hebron-george — APPROVED** [review]

**Pull request merged**